### PR TITLE
Feature/player disappear

### PR DESCRIPTION
**Pull request by MaximeGLegault**
Encapsuler de la logique de transformation des packets en image utilisable par kalman.
Refactor ImageKalmanTransformer
Can now stop all robots at once.

Some things that shouldn't be here also.

### CODE REVIEW

File: ai/executors/command_executor.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def exec(self) -> List[Command]:
         # Transform to other command type
         for player in self.ws.game_state.my_team.available_players.values():
             ready_to_ship_robot_packet_list += (self._parse_ai_command(player))
+            # player.set_command()
```
Comment (david2194) [inline ai/executors/command_executor.py]: Commentaire oublié?

File: RULEngine/Game/Game.py [hunk 2]
```diff
@@ -96,23 +96,13 @@ def _update_players(self, vision_frame, delta):
         self._update_players_of_team(yellow_team, self.yellow_team, delta)
 
     def kalman_update_ball(self, vision_frame, delta):
-        kalman_list = []
-        for c in vision_frame:
-            kalman_list.append(c["ball"])
-        self.ball.kalman_update(kalman_list, delta)
+        self.ball.kalman_update(vision_frame["ball"], delta)
```
Comment (sim0504) [inline RULEngine/Game/Game.py]: pourquoi on prend pas le self.delta_t de Game ?

File: ai/executors/command_executor.py (hunk 1 above)
Comment (sim0504) [inline ai/executors/command_executor.py]: uncomment

File: RULEngine/Game/Game.py (hunk 2 above)
Comment (MaximeGLegault) [inline RULEngine/Game/Game.py]: It is the same. We could reduce the signature of the method by using it though. Will change.

**Comment by MaximeGLegault** [comment]
It has been fully tested in real life.

**Pull request merged**